### PR TITLE
Bug 1724454: example VLAN configurations

### PR DESCRIPTION
**Pull request by miabbott**
Provides examples for configuring VLANs on the kernel command line
when installing RHCOS.

### CODE REVIEW

**Comment by miabbott** [comment]
cc: @dustymabe @sagidlow @bobfuru

**Comment by miabbott** [comment]
This PR caused a horizontal scrollbar to appear in the table I changed, but previously similarly long lines in the table did not have any scroll bar.  Not sure what is going on here.

If this is approved, this probably needs to be cherry-picked all the way back to 4.6

File: modules/installation-user-infra-machines-static-network.adoc [hunk 1]
```diff
@@ -30,6 +30,9 @@ Ordering is important when adding kernel arguments: `ip=`, `nameserver=`, and th
 ====
 
 .Routing and bonding options for ISO
+
+The table below provides examples for configuring networking of your {op-system-first} nodes.  These are networking options that are passed to `dracut` during the boot of the system.  For more information about the networking options supported by `dracut`, please see the manpage for `dracut.cmdline`.
```
Comment (bobfuru) [inline modules/installation-user-infra-machines-static-network.adoc:34]:
```suggestion
The following table provides examples for configuring networking of your {op-system-first} nodes. These are networking options that are passed to the `dracut` tool during system boot. For more information about the networking options supported by `dracut`, see the `dracut.cmdline` manual page.
```
These are suggestions based on IBM Style Guide suggestions and interestingly a recent docs team conversation about `man page` vs. `manual page` sparked by `kdump` work from @kelvinfan001. :)

File: modules/installation-user-infra-machines-static-network.adoc [hunk 2]
```diff
@@ -70,6 +73,22 @@ ip=enp1s0:dhcp
...
+a|Optional: You can configure VLANs on individual interfaces by using the `vlan=` parameter.
+a| 
+To configure a VLAN on a network interface and use a static IP address:
+
+----
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0.100:none
+vlan=enp2s0.100:enp2s0
+----
+
+To configure a VLAN on a network interface and use DHCP:
```
Comment (bobfuru) [inline modules/installation-user-infra-machines-static-network.adoc:85]:
```suggestion
To configure a VLAN on a network interface and to use DHCP:
```
Matching the wording you use in the next example on l129.

**Review by bobfuru — COMMENTED** [review]
Thanks for submitting this, @miabbott! I have a few minor suggested edits that you can manually apply if you agree. (IIUC, accepting the changes in GH adds an additional commit that you'd have to squash to meet our docs DoD.)
This is a fantastic improvement!

Oh, and regarding the scrollbar...I tried a local asciibinder build and didn't see the issue showing using that method. Formatting in the .adoc looks fine. Fingers crossed that it's just an issue with the netlify preview build.

**Review by dustymabe — APPROVED** [review]

**Comment by miabbott** [comment]
@bobfuru Thanks for the suggestions; squashed and pushed a new commit :arrow_up:

**Comment by bobfuru** [comment]
LGTM

**Pull request merged**